packages/client/test/integration/mocks/mockchain.ts

```diff
@@ -23,16 +24,22 @@ export default class MockChain extends Chain {
 
   async build() {
     const blocks: Block[] = []
+    const mergeBlockNumber = this.config.chainCommon.hardforkBlockBN('merge')?.toNumber();
     for (let number = 0; number < this.height; number++) {
       const block = Block.fromBlockData({
         header: {
           number: number + 1,
-          difficulty: 1,
+          difficulty: number > 10 ? 0 : new BN(1),
```

Comment:
i would imagine this would break the other tests?

Reply:
Yeah, I need to change that hardcoded 10 to be the block number for the merge hardfork.